repo2docker/buildpacks/python/__init__.py

```diff
@@ -146,6 +155,8 @@ def detect(self):
                 return True
             else:
                 return False
+        if not os.path.exists('binder') and os.path.exists(setup_py):
+            return True
```

Comment:
Does this mean that if a repo *doesn't* have a `binder/` folder, nor a `requirements.txt`, but *does* have a `setup.py` file then it'll still be built?

Reply:
yes, i.e. pointing Binder to a typical, basic Python package.